e2e/README.md

```diff
@@ -22,15 +13,8 @@ Now that the polygon-edge binary is in the `go/bin` folder, the e2e test server
 If you've stopped the tests abruptly, chances are the polygon-edge process is still running on your machine. <br/ >
 In order for the tests to function normally, please kill the process.
 
-### Check if the polygon-edge-* folders are present in /tmp/
-
-While running, the e2e server stores data needed for running in the /tmp/polygon-edge* folders. <br />
-To clean up these folders, simply run:
-
-````bash
-cd /tmp/
-rm -rf polygon-edge-*
-````
+Note the process id: `ps aux | grep polygon-edge`
+Kill the process `kill -9 [pid]`
```

Comment:
Could we implement this in the e2e framework itself or through some command, instead of doing it manually?

Reply:
Adapted docs, WDYT?